admin/services/data-access/pupil-restart.data.service.js

```diff
@@ -115,20 +115,80 @@ pupilRestartDataService.sqlFindLatestRestart = async function (pupilId) {
 
 /**
  * Find pupil's restart codes
- * @param pupilId
  * @return {Promise.<void>}
  */
 pupilRestartDataService.sqlFindRestartCodes = async function () {
   const sql = `
   SELECT 
     id, 
     code, 
-    statusDesc 
+    description
   FROM ${sqlService.adminSchema}.[pupilRestartCode]
-  ORDER BY statusDesc ASC`
+  ORDER BY description ASC`
   return sqlService.query(sql)
 }
 
+/**
+ * Find pupil's restart reason description by id
+ * @param id
+ * @return {Promise.<void>}
+ */
+pupilRestartDataService.sqlFindRestartReasonDescById = async function (id) {
+  console.log('fdsfdsdsfdsfdsfasd', id)
```

Comment:
please remove this, or change the message and use winston

Reply:
@GuyHarwood removed